elasticms-admin/config/packages/security.yaml

```diff
@@ -77,6 +77,7 @@ security:
         - { path: ^/resetting, role: PUBLIC_ACCESS }
         - { path: ^/data/file, role: PUBLIC_ACCESS }
         - { path: ^/data/asset, role: PUBLIC_ACCESS }
+        - { path: ^/channel/.*/emsch/api, role: ROLE_API }
```

Comment:
Is this really needed? the other roles should do the job? What about a CMS where the bridge is used only in the channel?

Reply:
But this is the admin security acl. Without this and with with a public channel the api is callable. You will get 403 without a token... need to think about it 😃

Reply:
No the bridge inside the channel calls directly the service layer, no 403 or token. So yes this is needed ...

Reply:
So, to me, it should be a ROLE_USER. Up to the data controllers to validate CT's role then

Reply:
And you ensure that your are connected

Reply:
Or a ROLE_AUTHOR, we should check. I think it's the minimum role to edit a document